client/src/components/composite/Admin/AdminAvailabilityView/AdminAvailabilityView.tsx

```diff
@@ -123,23 +124,23 @@ const AdminAvailabilityView = ({
             selectRange
             value={
               dateRangeDefined
-                ? [
-                    new Date(startDate.seconds * MS_IN_SECOND),
-                    new Date(endDate.seconds * MS_IN_SECOND)
-                  ]
+                ? [timestampToDate(startDate), timestampToDate(endDate)]
                 : undefined
             }
-            tileContent={({ date }) =>
-              // Find slots that are "available"
-              slots.some(
+            tileContent={({ date }) => {
+              const slot = slots.find(
+                // Find slots that are "available"
                 (slot) =>
-                  new Date(slot.date.seconds * MS_IN_SECOND).toDateString() ===
+                  timestampToDate(slot.date).toDateString() ===
                     date.toDateString() && slot.maxBookings > 0
```

Comment:
Can we actually remove slot.maxbookings > 0

Reply:
resolved